src/view/pane/room.js

```diff
@@ -145,8 +141,7 @@ Candy.View.Pane = (function(self, $) {
            *   (String) roomJid - Room JID
            *   (jQuery.Element) element - Room element
            */
-          $(Candy).triggerHandler('candy:view.room.after-show', evtData);
-
+          $(Candy).triggerHandler('candy:view.room.after-show', roomObject);
```

Comment:
Even for a single item? I purposefully ditched it because it seemed silly when there was only one expected data object.

Actually, just talked myself out of it. It makes me it more flexible in the future.  I'll change to that, and yes--documentation. :)